public/pages/CreateMonitor/components/AnomalyDetectors/AnomaliesChart/AnomaliesChart.js

```diff
@@ -0,0 +1,134 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EuiText, EuiSpacer } from '@elastic/eui';
+import {
+  Chart,
+  getAxisId,
+  Axis,
+  getSpecId,
+  LineSeries,
+  niceTimeFormatter,
+  Settings,
+  Position,
+  getAnnotationId,
+  LineAnnotation,
+} from '@elastic/charts';
+import { ChartContainer } from '../../../../../components/ChartContainer/ChartContainer';
+import DelayedLoader from '../../../../../components/DelayedLoader';
+
+const getAxisTitle = (displayGrade, displayConfidence) => {
```

Comment:
minor: return null if both of them are false.

Reply:
If both params are false, we should return null. Currently our code will always set one of them as true.